src/awscdk/github.ts

```diff
@@ -69,7 +82,14 @@ export class GithubCDKPipeline extends CDKPipeline {
     });
 
     // Determine if versioned artifacts are necessary.
-    this.needsVersionedArtifacts = this.options.stages.find(s => s.manualApproval === true) !== undefined;
+    this.needsVersionedArtifacts = options.stages.find(s => s.manualApproval === true) !== undefined;;
+    this.useGithubPackages = this.needsVersionedArtifacts && (options.useGithubPackagesForAssembly ?? false);
```

Comment:
Why is the default false? Should we move the default to "true"?

Reply:
For backwards compatibility reasons